2025-03-05 03:29:54,463 INFO: Starting application in development mode [in F:\PythonII\Dating\app\utils\logging.py:43]
2025-03-05 03:29:55,262 INFO: Admin user already exists. [in F:\PythonII\Dating\app\__init__.py:24]
2025-03-05 03:29:55,283 WARNING:  * Debugger is active! [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
2025-03-05 03:29:55,296 INFO:  * Debugger PIN: 752-258-082 [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
2025-03-05 03:29:58,485 INFO: 127.0.0.1 - - [05/Mar/2025 03:29:58] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
2025-03-05 03:29:58,672 INFO: 127.0.0.1 - - [05/Mar/2025 03:29:58] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
2025-03-05 03:29:58,702 INFO: 127.0.0.1 - - [05/Mar/2025 03:29:58] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
2025-03-05 03:29:58,886 INFO: 127.0.0.1 - - [05/Mar/2025 03:29:58] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
2025-03-05 03:29:59,082 INFO: 127.0.0.1 - - [05/Mar/2025 03:29:59] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\muhammad.ziada\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py:96]
